descriptors/ModuleDescriptor-template.json

```diff
@@ -252,7 +256,9 @@
             "instance-authority-links.authority-statistics.collection.get",
             "configuration.entries.collection.get",
             "configuration.entries.item.post",
-            "configuration.entries.item.delete"
+            "configuration.entries.item.delete",
+            "mod-settings.entries.item.get",
```

Comment:
it's incorrect permission, plz review https://github.com/folio-org/mod-finance/pull/295/files

Reply:
not sure you even need to update _tenant permissions, timezone is requested during circ log export, seems you need to update system user permissions in mod-data-export-spring